src/integrationtests/java/com/aws/greengrass/integrationtests/deployment/SubGroupDeploymentIntegrationTest.java

```diff
@@ -251,6 +329,24 @@ private void verifyServicesRemoved(String... services) {
         }
     }
 
+    private void verifyRejectionStatus(String... groups) {
```

Comment:
Is there a way to verify that the Rejected status is reported to Jobs, since that also goes through the MQTT client similar to FSS updates? Not blocking this PR for that but if it should be tested somewhere, in a UAT would be fine too if that's easier

Reply:
Capturing jobs status was little tricky, but I have added it.